app/connectors/VatReturnHttpParser.scala

```diff
@@ -39,10 +39,10 @@ object VatReturnHttpParser extends Logging {
               Left(InvalidJson)
           }
         case NOT_FOUND =>
-          logger.warn("Received NotFound from vat return")
+          logger.warn("Received NotFound for vat return")
           Left(NotFound)
         case CONFLICT =>
-          logger.warn("Received NotFound from vat return")
+          logger.warn("Conflict found for vat return")
```

Comment:
Could we make this 'Received Conflict for vat return' just for consistency

Reply:
Done